Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM scratch
```

Comment:
Use a [multistage build](https://docs.docker.com/develop/develop-images/multistage-build/) and build this inside of Docker entirely.

Reply:
Something like this should work,
```dockerfile
FROM golang AS gobuilder
RUN go get -u github.com/jmhobbs/terminal-parrot

FROM scratch
COPY --from=gobuilder /go/bin/terminal-parrot /terminal-parrot
CMD ["/terminal-parrot"]
```

Though that totally ignores the working dir and gets it from the repo on build.  Would also want a dockerignore in that case.

Reply:
I wouldn't do `go get` -- I'd just use the current working directory and build.

Reply:
I'm chuckling to myself because multistage builds didn't exist when I first submitted the PR.  :-D

Reply:
See the docs I linked -- use the working directory.

Reply:
@pdevine well they do now and it's not that difficult to handle :)

Reply:
@robbyoconnor no argument there.